server/models.py

```diff
@@ -404,14 +405,24 @@ class Batch(BaseModel):
         order_by="BatchResultTallySheet.created_at",
     )
 
-    # Match our session fields
-    last_edited_by_user_type = Column(String(200))
-    last_edited_by_user_key = Column(
-        String(200)
-    )  # An email for jurisdiction admins, a user ID for tally entry users
+    # Only one of the following should be populated, enforced via a check constraint defined below
     last_edited_by_support_user_email = Column(String(200))
+    last_edited_by_jurisdiction_admin_email = Column(String(200))
```

Comment:
I wonder if this should be a foreign key to the `User` table now

Reply:
Assuming a foreign key on user ID*, I went back and forth on this. Because the session contains user email, and that's ultimately what we want to display, it initially felt roundabout to retrieve an ID given email at write time and convert back from ID to email at read time.

That said, consistency in how we handle JAs and tally entry users would be nice! And a foreign key on user ID is probably what we'll want if/when we have a universal users table. I ultimately did decide to switch for these reasons.

https://github.com/votingworks/arlo/pull/1673/commits/2b75cbba2c82e93b4a7b8d868ffa0f6319c5c0d9

\*I know we could also set up a foreign key on user email since it's unique, but user ID felt ideal for the other reasons outlined above